acceptance/gossip_peerings_test.go

```diff
@@ -148,13 +160,21 @@ func TestGossipRestart(t *testing.T) {
 	checkRangeReplication(t, l, 10*time.Second)
 
 	log.Infof("stopping all nodes")
-	for _, node := range l.Nodes {
+	for i, node := range l.Nodes {
 		node.Stop(5)
+		expEvents = append(expEvents, []localcluster.Event{
+			{NodeIndex: i, Status: "kill"},
```

Comment:
I've noticed that event notification has changed across docker releases. I think we need to be careful in not being overly strict in what we expect. The current `AssertAndStop` API might not be appropriate given the constraints we're under.